stable/stolon/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: stolon
-version: 1.5.9
+version: 1.6.0
```

Comment:
Next version should be 1.5.10

Reply:
I changed it to 1.6 as this is a possibly breaking change: If someone worked around the broken behavior their deploy would break. Or, if they set the clusterName to one thing but ended up with fullName then switching to clusterName under their feet could cause problems as well.

Reply:
ok, please rebase the PR and I'll approve

Reply:
This has been rebased